Source/items.cpp

```diff
@@ -1773,7 +1777,7 @@ void SaveItemPower(int i, item_effect_type power, int param1, int param2, int mi
 		break;
 	case IPL_DOPPELGANGER:
 		items[i]._iDamAcFlags |= 16;
-		// no break
+		// fall through

```

Comment:
Use the C++17 attribute here and below:

```suggestion
		[[fallthrough]];
```